packages/next/src/experimental/testing/server/utils.ts

```diff
@@ -4,6 +4,16 @@ import { NodeNextRequest } from '../../../server/base-http/node'
 import type { BaseNextRequest } from '../../../server/base-http'
 import type { NextResponse } from '../../../server/web/exports'
 
+function parseUrl(url: string): URL | null {
```

Comment:
You can use https://github.com/vercel/next.js/blob/15165727a477159abce7b945d8677f34d6dff22e/packages/next/src/lib/url.ts#L9

Reply:
Done, thank you!